src/Models/BaseElement.php

```diff
@@ -1261,11 +1261,12 @@ public function TotalItems()
     /**
      * Returns the position of the current element.
      *
+     * @param int $startIndex Number to start count from.
      * @return int
      */
-    public function Pos()
+    public function Pos($startIndex = 1)
```

Comment:
I appreciate the initiative, but unfortunately we can't add new parameters (even optional ones) to public API outside of a major release as per the [major release policy](https://docs.silverstripe.org/en/5/project_governance/major_release_policy/) and [definition of public API](https://docs.silverstripe.org/en/5/project_governance/public_api/).

We could accept this in CMS 6, so feel free to submit a new pull request targetting the `6` branch - but I'll close this one.

Reply:
Thanks for the feedback. It's disappointing that the policy is to class optional parameters as a breaking change. However if that's the policy then fair enough I'll add another PR.